src/webrtc.ts

```diff
@@ -89,6 +89,7 @@ export class WebrtcExtension extends EventEmitter implements StreamingClientExte
   private iceServers: RTCIceServer[] = [];
   private stanzaInstance?: NamedAgent;
   private webrtcSessions: GenesysCloudMediaSession[] = [];
+  private previousReinvite: string | undefined = '';
```

Comment:
I had a couple of thoughts today:

1. Do we need to support multiple sessions being re-invited? I imagine the most likely scenario for this is someone with an active call and a call on hold. I don't think we support re-invite for softphone right now, though, so maybe it doesn't matter at this point.
2. Do you think it'd be worth clearing it when we establish the session? I don't think it's a big deal since it's just a single ID that will be overwritten, but it seems like the "right" thing to do. 🙂